lib/cirro_io_v2/resources/gig.rb

```diff
@@ -0,0 +1,18 @@
+module CirroIOV2
+  module Resources
+    class Gig < Base
+      def create(title:, description:, url:, start_at:, end_at:, total_seats:, invitation_mode:, filter_query:,
+                 tasks:, notification_payload: nil, epam_options: nil)
+        # TODO: Is it ok? as rubocop says:  Avoid parameter lists longer than 5 parameters.
+        # [6/5] (https://rubystyle.guide#too-many-params)
+
+        body = { title: title, description: description, url: url, start_at: start_at, end_at: end_at,
+                 total_seats: total_seats, invitation_mode: invitation_mode, filter_query: filter_query, tasks: tasks,
+                 notification_payload: notification_payload, epam_options: epam_options }
+        # TODO: should I have some validations for body params?
+        response = client.request_client.request(:post, resource_root, body: body)
+        OpenStruct.new(response.body)
```

Comment:
I was actually hoping to have our own response classes e.g. Responses::Gig which can be a struct of course.
Also please don't use OpenStruct, if possible always use Struct instead.
https://www.rubydoc.info/gems/rubocop/0.61.1/RuboCop/Cop/Performance/OpenStruct